rfcs/0028-nix-release-model.md

```diff
@@ -0,0 +1,159 @@
+---
+feature: nix-release-model
+start-date: 2018-03-28
+author: Shea Levy <shea@shealevy.com>
+co-authors: Michael Raskin <7c6f434c@mail.ru> (any other Nix committers on-board?)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+A model of when, what, and how to make new releases of Nix.
+
+# Motivation
+[motivation]: #motivation
+
+We want to set clear expectations for users, provide guidance for
+developers, and provide both groups the benefits of easy availability
+of new improvements.
+
+# Detailed design
+[design]: #detailed-design
+
+The proposed process contains a number of interrelated components.
+I've numbered them for ease of reference and discussion, not to
+indicate any meaningful ordering. Any aspect of this can be suspended
+at the security team's discretion when handling a vulnerability.
```

Comment:
Maybe we want to follow the good part of Linus Torvalds' position on security bugs and say that the same is expected to happen in case of data-loss bugs…

Reply:
Fixed.